src/App.css

```diff
@@ -55,178 +55,181 @@ body {
   position: absolute;
   bottom: 10%;
 }
-.degree-span-row-2{
+.degree-span-row-2 {
   grid-row: span 2 / auto;
 }
 
-.left h1{
-    margin: 0;
+.left h1 {
+  margin: 0;
 }
 
 html {
-    height: 100%;
+  height: 100%;
 }
 
 body {
-    min-height: 100vh;
+  min-height: 100vh;
 }
 
 /* -----------------------
     Form
    ---------------------- */
 
 .search-form {
-    position: relative;
-    background: black;
-    float: right;
-    height: 100%;
-    border-bottom: 2px solid white;
-    overflow: hidden;
-    width: 100%;
+  position: relative;
+  background: black;
+  float: right;
+  height: 100%;
+  border-bottom: 2px solid white;
+  overflow: hidden;
+  width: 100%;
 }
 
 .search-form form input {
-    width: 70%;
-    min-height: 10vh;
-    border: none;
-    border-bottom: 2px solid white;
-    margin-left: 30px;
-    background: transparent;
-    font-size: 20px;
-    color: white;
-    outline: none;
+  width: 70%;
+  min-height: 10vh;
+  border: none;
+  border-bottom: 2px solid white;
+  margin-left: 30px;
+  background: transparent;
+  font-size: 20px;
+  color: white;
+  outline: none;
 }
 
 .search-form form button {
-    margin-top: 20px;
-    width: 15%;
-    height: auto;
-    background: black;
-    float: right;
-
+  margin-top: 20px;
+  width: 15%;
+  height: auto;
+  background: black;
+  float: right;
 }
 
 .search-form form button img {
-    width: 100%;
-    height: auto;
-    background: white;
-    float: right;
-
+  width: 100%;
+  height: auto;
+  background: white;
+  float: right;
 }
 
 .search-form .popular-searches {
-    margin-top: 30px;
-    margin-left: 30px;
+  margin-top: 30px;
+  margin-left: 30px;
 }
 
 .search-form .popular-searches .popular-data {
-    margin-bottom: 10px;
-    color: white;
-    font-size: 20px;
-    letter-spacing: 2px;
+  margin-bottom: 10px;
+  color: white;
+  font-size: 20px;
+  letter-spacing: 2px;
 }
 
 .loader {
-    text-align: center;
-    margin-top: 200px;
+  margin-left: 650px;
+  margin-top: 200px;
```

Comment:
I have a feeling that you should use  % instead px  so that it would be uniform with every screen.

Reply:
Thanks @hezzie 
This issue has been solved, you can check it out

Reply:
Nice work @victorkarangwa